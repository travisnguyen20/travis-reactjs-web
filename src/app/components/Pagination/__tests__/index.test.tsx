import * as React from 'react';
import { render } from '@testing-library/react';

import { Pagination } from '..';

describe('<Pagination  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Pagination />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
