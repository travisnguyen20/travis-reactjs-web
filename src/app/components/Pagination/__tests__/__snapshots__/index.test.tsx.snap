// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pagination  /> should match snapshot 1`] = `
<ul
  class="pagination"
>
  <li
    class="page-item disabled"
  >
    <a
      aria-disabled="true"
      aria-label="Previous page"
      class="page-link"
      rel="prev"
      role="button"
      tabindex="0"
    >
      Previous
    </a>
  </li>
  <li
    class="page-item active"
  >
    <a
      aria-current="page"
      aria-label="Page 1 is your current page"
      class="page-link"
      role="button"
      tabindex="0"
    >
      1
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 2"
      class="page-link"
      role="button"
      tabindex="0"
    >
      2
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 3"
      class="page-link"
      role="button"
      tabindex="0"
    >
      3
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 4"
      class="page-link"
      role="button"
      tabindex="0"
    >
      4
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 5"
      class="page-link"
      role="button"
      tabindex="0"
    >
      5
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 6"
      class="page-link"
      role="button"
      tabindex="0"
    >
      6
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      class="page-link"
      role="button"
      tabindex="0"
    >
      ...
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-label="Page 100"
      class="page-link"
      role="button"
      tabindex="0"
    >
      100
    </a>
  </li>
  <li
    class="page-item"
  >
    <a
      aria-disabled="false"
      aria-label="Next page"
      class="page-link"
      rel="next"
      role="button"
      tabindex="0"
    >
      Next
    </a>
  </li>
</ul>
`;
